sdktests/common_tests_tags.go

```diff
@@ -207,10 +206,20 @@ func (c CommonTagsTests) makeValidTagsTestParams() []tagsTestParams {
 		o.None[string](),
 		o.Some(""), // empty string
 	}
-	for i := 0; i < len(allAllowedTagChars); i += maxTagValueLength {
```

Comment:
This test is trying to test all valid characters, but it was really just testing that we would ignore spaces. 

Now it will actually test the right stuff.